### PR TITLE
Putting eVoucher in list for phone vouchers

### LINKED ISSUE #1559: Show eVoucher instead of voucher ID for non-paper vouchers

### PR DESCRIPTION
**Pull request by manuelplazaspalacio**
Close #1559

### CODE REVIEW

File: app/src/ereferrals/java/org/eyeseetea/malariacare/layout/adapters/dashboard/strategies/DashboardAdapterStrategy.java [hunk 1]
```diff
@@ -95,5 +101,17 @@ public void renderSurveySummary(View rowView, SurveyDB survey) {
     public boolean hasAllComplementarySurveys(SurveyDB malariaSurvey) {
         return true;
     }
+
+    private boolean hasPhone(SurveyDB survey) {
```
Comment (ifoche) [inline app/src/ereferrals/java/org/eyeseetea/malariacare/layout/adapters/dashboard/strategies/DashboardAdapterStrategy.java:105]: @manuelplazaspalacio maybe we could improve the performance of this by using a join? Is there any reason for not doing it?

**Review by ifoche — CHANGES_REQUESTED** [review]
@manuelplazaspalacio just a minor change in how you retrieve the value from the DB and loop on the values

**Comment by manuelplazaspalacio** [comment]
Refactor has phone method @ifoche

**Review by ifoche — APPROVED** [review]
thanks @manuelplazaspalacio

**Pull request merged**